Orion identity: product name, basegame, home path and config prefix.
Touches cmake/identity.cmake, the #ifdef STANDALONE block of
code/qcommon/q_shared.h, and cmake/renderer_common.cmake so that the
dlopen'd renderer (which creates the window) also sees STANDALONE.
Upstream formatting is left untouched.
Base: ioquake3 588393618dbc82e7207c21c6ddecca229944a03a

diff --git a/cmake/identity.cmake b/cmake/identity.cmake
index e90934c7..ba7862f8 100644
--- a/cmake/identity.cmake
+++ b/cmake/identity.cmake
@@ -1,10 +1,10 @@
-set(PROJECT_NAME ioq3)
-set(PROJECT_VERSION 1.36)
+set(PROJECT_NAME orion)
+set(PROJECT_VERSION 0.1.0)
 
-set(SERVER_NAME ioq3ded)
-set(CLIENT_NAME ioquake3)
+set(SERVER_NAME orionded)
+set(CLIENT_NAME orion)
 
-set(BASEGAME baseq3)
+set(BASEGAME reach)
 
 set(CGAME_MODULE cgame)
 set(GAME_MODULE qagame)
@@ -13,9 +13,9 @@ set(UI_MODULE ui)
 set(WINDOWS_ICON_PATH ${CMAKE_SOURCE_DIR}/misc/windows/quake3.ico)
 
 set(MACOS_ICON_PATH ${CMAKE_SOURCE_DIR}/misc/macos/quake3_flat.icns)
-set(MACOS_BUNDLE_ID org.ioquake.${CLIENT_NAME})
+set(MACOS_BUNDLE_ID org.73nn0.${CLIENT_NAME})
 
-set(COPYRIGHT "QUAKE III ARENA Copyright © 1999-2000 id Software, Inc. All rights reserved.")
+set(COPYRIGHT "Copyright © 2026 73NN0")
 
-set(CONTACT_EMAIL "info@ioquake.org")
-set(PROTOCOL_HANDLER_SCHEME quake3)
+set(CONTACT_EMAIL "orion@oni.reach")
+set(PROTOCOL_HANDLER_SCHEME orion)
diff --git a/cmake/renderer_common.cmake b/cmake/renderer_common.cmake
index 277bf46a..51abea73 100644
--- a/cmake/renderer_common.cmake
+++ b/cmake/renderer_common.cmake
@@ -27,6 +27,12 @@ if(USE_FREETYPE)
     list(APPEND RENDERER_DEFINITIONS BUILD_FREETYPE)
 endif()
 
+# The dlopen'd renderer compiles sdl_glimp.c: without this, the window
+# title and other STANDALONE identity values fall back to ioquake3.
+if(BUILD_STANDALONE)
+    list(APPEND RENDERER_DEFINITIONS STANDALONE)
+endif()
+
 if(USE_RENDERER_DLOPEN)
     list(APPEND RENDERER_DEFINITIONS USE_RENDERER_DLOPEN)
 elseif(BUILD_RENDERER_GL1 AND BUILD_RENDERER_GL2)
diff --git a/code/qcommon/q_shared.h b/code/qcommon/q_shared.h
index 1877a42c..7a0abc43 100644
--- a/code/qcommon/q_shared.h
+++ b/code/qcommon/q_shared.h
@@ -27,18 +27,18 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 // A user mod should never modify this file
 
 #ifdef STANDALONE
-  #define PRODUCT_NAME				"iofoo3"
-  #define BASEGAME					"foobar"
-  #define CLIENT_WINDOW_TITLE		"changeme"
-  #define CLIENT_WINDOW_MIN_TITLE	"changeme2"
-  #define HOMEPATH_NAME_UNIX_LEGACY	".foo"
-  #define HOMEPATH_NAME				"FooBar"
-  #define GAMENAME_FOR_MASTER		"foobar"	// must NOT contain whitespace
-  #define CINEMATICS_LOGO		"foologo.roq"
+  #define PRODUCT_NAME				"orion"
+  #define BASEGAME					"reach"
+  #define CLIENT_WINDOW_TITLE		"orion"
+  #define CLIENT_WINDOW_MIN_TITLE	"orion"
+  #define HOMEPATH_NAME_UNIX_LEGACY	".orion"
+  #define HOMEPATH_NAME				"orion"
+  #define GAMENAME_FOR_MASTER		"orion"	// must NOT contain whitespace
+  #define CINEMATICS_LOGO		"logo.roq"
   #define CINEMATICS_INTRO		"intro.roq"
 //  #define LEGACY_PROTOCOL	// You probably don't need this for your standalone game
 //  #define PROTOCOL_HANDLER		"foobar"
-  #define CONFIG_PREFIX			"fooconfig"
+  #define CONFIG_PREFIX			"orion"
 #else
   #define PRODUCT_NAME				"ioq3"
   #define BASEGAME					"baseq3"
